lua/wikis/valorant/MatchPage.lua

```diff
@@ -142,7 +140,27 @@ function MatchPage:_renderGameOverview(game)
 		}
 	end
 
-	return Div{
+	local function createScoreHolder()
+		return Div{
+			classes = {'match-bm-lol-game-summary-score-holder'},
+			children = game.finished and WidgetUtil.collect(
+				not self:isBestOfOne() and Div{
```

Comment:
Non finished games should display score too

Reply:
idk, but `0-0` is how we show an ongoing match/map atm in the bracket/matchlist/matchsummaries, which is on purpose to visual difference it from a non-live match